src/main/java/com/kirinpatel/sync/gui/VLCJMediaPlayer.java

```diff
@@ -54,10 +54,10 @@
         media  = new Media("");

```

Comment:
Can we initialize the media object with a mediaplayer and then have the same "API" as the mediaplayer? So below, in the play method, you would just do media.play() and it would play the mediaplayer as well as set paused to false.